lxd/daemon.go

```diff
@@ -632,88 +658,89 @@ func startDaemon() (*Daemon, error) {
 		NotFound.Render(w)
 	})
 
```

Comment:
For now let's go with just the if statement, that should reduce the size of the diff. If at some point we need to shrink the function, we'll split it up then.